WalletWasabi.Fluent/Models/UiTriggers.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Reactive;
+using System.Reactive.Linq;
+using ReactiveUI;
+using WalletWasabi.Blockchain.TransactionProcessing;
+using WalletWasabi.Fluent.Extensions;
+using WalletWasabi.Fluent.ViewModels.Wallets;
+using WalletWasabi.Wallets;
+
+namespace WalletWasabi.Fluent.Models;
+
+public class UiTriggers
+{
+	private readonly WalletViewModel _walletViewModel;
+	private readonly Wallet _wallet;
+
+	public UiTriggers(WalletViewModel walletViewModel)
+	{
+		_walletViewModel = walletViewModel;
+		_wallet = _walletViewModel.Wallet;
+	}
+
+	/// <summary>
+	/// Triggers on subscription and when a transaction to the wallet is processed.
+	/// </summary>
+	public IObservable<Unit> WalletRelevantTransactionProcessed =>
+		Observable.FromEventPattern(_wallet.TransactionProcessor, nameof(TransactionProcessor.WalletRelevantTransactionProcessed)).ToSignal().StartWith(Unit.Default);
+
+	/// <summary>
+	/// Triggers on subscription and when the USD exchange rate changed.
+	/// </summary>
+	public IObservable<Unit> UsdExchangeRateChanged => _wallet.Synchronizer.WhenAnyValue(x => x.UsdExchangeRate).ToSignal();
+
+	/// <summary>
+	/// Triggers on subscription and when the anon score target changed.
+	/// </summary>
+	public IObservable<Unit> AnonScoreTargetChanged => _walletViewModel.CoinJoinSettings.WhenAnyValue(x => x.AnonScoreTarget).ToSignal().Throttle(TimeSpan.FromMilliseconds(3100));
```

Comment:
I'd be wary of putting Throttle in this class, unless you always want to throttle updates. If consumer **a** wants different throttling than consumer **b**, we will be in trouble.

Reply:
No, this is a must have.

In the wallet settings if the anonscore changes it only gets saved after 3 seconds. So if the throttle wasn't there, the subscribers would get the "It changed" sign before it would be actually saved. 

The only problem here is the throttle times are not unified software wide, but this is out of the scope of this PR.